src/main/java/org/spongepowered/api/text/selector/SelectorTypes.java

```diff
@@ -24,56 +24,33 @@
  */
 package org.spongepowered.api.text.selector;
 
-import com.google.common.base.Optional;
-
-import java.util.List;
-
 /**
- * SelectorTypes is a list of the default selector types that are
- * available in Vanilla Minecraft.
+ * SelectorTypes is a list of the default selector types that are available in
+ * Vanilla Minecraft.
  */
 public final class SelectorTypes {
 
+    private SelectorTypes() {
```

Comment:
Why did you move the constructor up there?
IIRC there are first the (static) fields and then the constructor.
In addition to that the constructor is not important at all for this class, so we should push it to the bottom as it has been before.